exercises/metric_conversion.livemd

```diff
@@ -96,9 +96,26 @@ defmodule Metric do
     iex> Metric.convert({:kilometer, 10}, :centimeter)
     1000000
   """
-  def convert(from, to) do
+  def convert({unit, amount}, to) do
+    case {unit, amount} do
+      {:kilometer, _} -> convert_to(amount * 1000, to)
+      {:meter, _} -> convert_to(amount, to)
+      {:centimeter, _} -> convert_to(amount * 0.01, to)
+      {:millimeter, _} -> convert_to(amount * 0.001, to)
+    end
+  end
+
+  def convert_to(meters, unit) do
```

Comment:
I would consider renaming this to make it easier to understand. `convert` and `convert_to` are a bit similar.

perhaps `from_meters` is a good name?